CriteoPublisherSdk/Sources/Network/CR_ApiQueryKeys.m

```diff
@@ -21,143 +21,149 @@
 
 @implementation CR_ApiQueryKeys
 
-+ (NSString *)appId {
++ (NSString *_Nonnull)appId {
```

Comment:
Same about using `NS_ASSUME_NONNULL_BEGIN`

Reply:
I have reverted the `_NonNull` specifier for all constants in CR_ApiQueryKeys and added `NS_ASSUME_NONNULL_BEGIN / NS_ASSUME_NONNULL_END`.
Resolving this conversation.